Jenkinsfile

```diff
@@ -602,7 +602,109 @@ pipeline {
                         """
                     }
                     steps{
-                        buildHipClangJob('/opt/rocm/llvm/bin/clang++', '', "", image+'-hip-clang', "/usr/local", cmd, "gfx908")
+                        buildHipClangJob('/opt/rocm/llvm/bin/clang++', '', "", image+'-hip-clang', "/usr/local", cmd, "gfx908", "default", "ON")
+                    }
+                }
+
+                stage('MIOpenTensile Half Hip Release') {
+                    agent{ label rocmnode("vega20") }
+                    environment{
```

Comment:
The MIOpenTensile stage still consists of 16 jobs. All 16 need to be replayed in case of a single error, which means waste of CI resources. Please split the MIOpenTensile stage to several stages, 3-4 jobs max in each.

[Informative] I am looking into CI logs right now.

Reply:
Will do once PR #674 is verified and merged